### PR TITLE
User class

### PR DESCRIPTION
**Pull request by mennoo1996**
Created user class for compability with the webserver

### CODE REVIEW

File: src/main/java/data/User.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     */
+    public User() {
+        this("", Role.NONE, null);
+        
+    }
+    
+    /**
+     * Enum for all the roles a user can have.
+     */
+    public static enum Role {
```
Comment (geenen124) [inline src/main/java/data/User.java]: For readability it would be nice if this wasn't placed among the instance methods. Maybe place it towards the top of the class?

File: src/main/java/data/ScriptingProject.java [hunk 2]
```diff
@@ -92,10 +99,20 @@ public ScriptingProject(String name, String description, double secondsPerCount)
...
     
     /**
+     * Add a user.
+     * @param user the user to add
+     */
+    public void addUser(User user) {
+        this.users.add(user);
+    }
+    
+
```
Comment (geenen124) [inline src/main/java/data/ScriptingProject.java:114]: Unnecessary whitespace

**Comment by geenen124** [comment]
Looks great!

**Comment by Markvanderuit** [comment]
Looks great, I'm gonna merge.
